### DISCUSSION TITLE
Active Fork of Goldendict ⇾ GoldenDict-NG

### DISCUSSION THREAD

**Discussion by shenlebantongying** [goldendict/goldendict#1622] [General]
You're welcomed to have a try:

GitHub: https://github.com/xiaoyifang/goldendict-ng
Documentation: https://xiaoyifang.github.io/goldendict-ng/

Improvements:

* Has automated up-to-date builds/releases for Linux/Windows/macOS
* Based on webengine (Chromium) instead of WebKit (which is no longer maintained)
* Supports Qt6 
* Bugs fixed
* New features
* Optimizations
* Cleaned up codebase for easier participating

None-coding aspects:

* Actually solving and hunting down years-old problems
* Friendly. You won't be publicly shamed for sending PRs.
* Follows the license and be an actual FOSS project just like the original
* No need to wait for certain PRs to be merged
* Malicious sabotagers and code thieves are blocked
* Accepts forking, if you love to, just fork goldendict-ng back to goldendict or platiumdict, and it will be totally OK, just keep the same license terms.

---

Not so related:

[the license of goldendict](https://github.com/goldendict/goldendict/blob/master/LICENSE.txt) guarantees [the essentials freedoms](https://www.gnu.org/philosophy/free-sw.html):

* The freedom to run the program as you wish, for any purpose (freedom 0).
* The freedom to **study how the program works, and change it so it does your computing as you wish** (freedom 1). Access to the source code is a precondition for this.
* The **freedom to redistribute copies so you can help others** (freedom 2).
* The **freedom to distribute copies of your modified versions to others (freedom 3). By doing this you can give the whole community a chance to benefit from your changes**. Access to the source code is a precondition for this[.](https://github.com/goldendict/goldendict/pull/1542)

**Comment by vedgy** [comment]
Users: beware of goldendict-ng's downsides (https://github.com/goldendict/goldendict/discussions/1616) and of deliberate one-sided advertisement on the part of goldendict-ng's developers (https://github.com/goldendict/goldendict/discussions/1618).

**Comment by vedgy** [comment]
This is not much of a discussion. More an advertisement. Other related discussions have more participants and actual conversation.
